commands/system.go

```diff
@@ -95,3 +105,18 @@ func clearBusyCmdF(c client.Client, cmd *cobra.Command, _ []string) error {
 	printer.PrintT("Busy state cleared", map[string]string{"status": "ok"})
 	return nil
 }
+
+func systemVersionCmdF(c client.Client, cmd *cobra.Command, _ []string) error {
+	printer.SetSingle(true)
+	// server version information comes with all responses. We can't
+	// use the initial "withClient" connection information as local
+	// mode doesn't need to log in, so we use an endpoint that will
+	// always return a valid response
+	_, resp := c.GetAllTeams("", 0, 0)
+	if resp.Error != nil {
+		return fmt.Errorf("unable to fetch teams information: %w", resp.Error)
```

Comment:
Would it be better to say "unable to fetch server version?"
```suggestion
		return fmt.Errorf("unable to fetch server version: %w", resp.Error)
```

Reply:
I've thought about this and still can't make my mind. It is true that we're fetching the server version, but only that error doesn't actually provide information on what request is really being done. It might be nonimportant tho, so I'm happy to apply the suggestion and come back to rethink this if it turns out to be a problem :slightly_smiling_face: